src/js/components/AppListStatusFilterComponent.jsx

```diff
@@ -33,18 +26,18 @@ var AppListStatusFilterComponent = React.createClass({
 
   getInitialState: function () {
     return {
-      appsStatusesCount: getInitialAppStatusCount(),
+      appsStatusesCount: {},
```

Comment:
Yeah, the name isn't maybe easy to read, but I think it's expressive.
Thank you for nitpicking anyways.

app_s_Status_es_Count -> I count the different status_es_ on my list of apps.

But now nitpicking extended from my side, haha. :)
appStatusCounters -> Counter is more an object that is doing something, not a simple number, imho. Status is singular, but I have multiple statuses.
appStatuses -> that only implies that I have multiple statuses, but not how many of each.